### DISCUSSION TITLE
language configuration in next.config.js messes up styling

### DISCUSSION THREAD

**Discussion by ufelgen** [neuefische/questions#438] [Web] [answered]
Hi everyone,
I have a weird issue. I tested for a11y with lighthouse and got the notification `<html> element does not have a [lang] attribute` . I fixed this by adding `module.exports = {
  i18n: {
    locales: ["en"],
    defaultLocale: "en",
  },
};`
this only works upon server restart (CTRL + C and npm run dev) but then at the same time I get the red error in the console:
<img width="762" alt="Bildschirmfoto 2023-01-05 um 13 54 00" src="https://user-images.githubusercontent.com/115288914/210784937-42181556-cea7-4481-ad0f-eefcad7de41c.png">
which messes up my styling (with styled components).
any ideas how to fix it?

Thanks!

**Comment by ufelgen** [comment]
ah never mind, I fixed it. I had replaced `module.exports = nextConfig;` with the code above but it's obviously still needed.

solution: 
`(module.exports = nextConfig),
  {
    i18n: {
      locales: ["en"],
      defaultLocale: "en",
    },
  };`

**Comment by ufelgen** [comment]
okay I am opening this question again. Apparently the solution above did not fix it all. with the first option I get the language attribute but not the formatting (I'm guessing because of the missing nextConfig) but with the second option the formatting is correct again but the language attribute is gone. 

Please help me export both!! I tried many different ways and none of them work.... Thanks!!

**Comment by actyralf** [comment, marked as answer]
Did you try this: `module.exports = { ...nextConfig, i18n: { locales: ["en"], defaultLocale: "en" }`

**Reply by ufelgen** [reply]
Thanks, this works perfectly!!

**Reply by Elgin-Schultz** [reply]
module.exports = {
  ...nextConfig,
  i18n: { locales: ["en"], defaultLocale: "en" },
};

At the end please add: , };
